src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/ElementsTable.java

```diff
@@ -1208,40 +1203,25 @@ static class ModifierFilter {
 
             AccessKind accessValue = null;
             for (ElementKind kind : ALLOWED_KINDS) {
-                switch (kind) {
-                    case METHOD:
-                        accessValue  = options.showMembersAccess();
-                        break;
-                    case CLASS:
-                        accessValue  = options.showTypesAccess();
-                        break;
-                    case PACKAGE:
-                        accessValue  = options.showPackagesAccess();
-                        break;
-                    case MODULE:
-                        accessValue  = options.showModuleContents();
-                        break;
-                    default:
-                        throw new AssertionError("unknown element: " + kind);
-
-                }
+                accessValue = switch (kind) {
```

Comment:
It feels awkward when adjacent `switch` statements use different formatting.

Reply:
Correction: `switch` _expressions_ in this case.

Reply:
will examine

Reply:
fixed